payloads/v10/channel.ts

```diff
@@ -27,12 +24,20 @@ export interface APIPartialChannel {
 	 * @see {@link https://discord.com/developers/docs/resources/channel#channel-object-channel-types}
 	 */
 	type: ChannelType;
+}
+
+export interface APINameChannel {
```

Comment:
```suggestion
export interface APINamedChannel {
```

Or APINamableChannel?